website/client/assets/scss/form.scss

```diff
@@ -85,8 +85,8 @@ input, textarea, input.form-control, textarea.form-control {
     &.input-group-icon {
       border: solid 1px $gray-400;
       border-right: none;
-      border-top-right-radius: 0px;
-      border-bottom-right-radius: 0px;
+      border-top-right-radius: 0;
+      border-bottom-right-radius: 0;
```

Comment:
I'm sort of new to this repo, so forgive me if I'm missing something, but what's the point to this sort of change? This seems inconsistent with all the other times that a single value is used in this file.